dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZKServer.java

```diff
@@ -94,8 +112,9 @@ public void shutdown() {
      *
      * @param port The port to listen on
      */
-    public static void startLocalZkServer(final int port) {
-        String zkDataDir = System.getProperty("user.dir") +"/zookeeper_data";
+    public void startLocalZkServer(final int port) {
+        String zkDataDir = System.getProperty("user.dir") + ("".equals(prefix) ? prefix : ("/" + prefix)) + "/zookeeper_data";
+        logger.warn("The path of zk server exists");
```

Comment:
Hi,
Why introduce `logger.warn("The path of zk server exists");`?

Reply:
It just warn the previous data may be overwritten.